### ISSUE TITLE
Fix vuex/fees bug 

### ISSUE THREAD

**Issue by valiafetisov** [sidestream-tech/unified-auctions-ui#84]
### Goal
Fixed `[vuex] do not mutate vuex store state outside mutation handlers.`

### Context
After merging #80, we discovered a bug:

<img width="585" alt="Screenshot 2022-02-03 at 20 44 23" src="https://user-images.githubusercontent.com/3393626/152417465-1292713b-2e3d-4dd2-97f6-560dba6f41ba.png">
<img width="611" alt="Screenshot 2022-02-03 at 20 44 41" src="https://user-images.githubusercontent.com/3393626/152417471-62dc0f12-6599-4b11-b4d2-9505de136dd2.png">

### Tasks
- [x] Investigate and document
- [x] Fix

**Comment by benediktkaiser** [comment]
This issue is that we call: 

```js
dispatch('fees/setup', null, { root: true });
```
https://github.com/sidestream-tech/unified-auctions-ui/blob/main/frontend/store/auctions.ts#L140

I will remove this line and then it should work again.

**Comment by benediktkaiser** [comment]
The other issue was that we were trying to mutate the store directly. Before we have a `.map()` function with which we made a new Object, which we then enriched with the restarting state. 

Once I removed the enriching with Fees we no longer made a new object resulting in us trying to mutate the store with `.foreach`, leading to the error. I changed the foreach to a map function which I then exported and everything is now working as intended!
